### PR TITLE
Fix: `make proto-gen` work 

### LINKED ISSUE #339: `make proto-gen` is broken
* need to fix `make proto-gen` to run upgraded proto builder docker image
* we should use similar proto config + builder images across the different projects (hub, rdk)

### PR DESCRIPTION
**Pull request by hoangdv2429**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

There is a problem with newest proto-gen in #347 so we should have this one in the meantime.

Closes #339

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

For Author:

- [ ]  Targeted PR against correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to Github issue with discussion and accepted design
- [ ]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

**Comment by mtsitrin** [comment]
`proto-gen` worked fine for me

`proto-format` failed:

```
Formatting Protobuf files
/bin/sh: -c: line 0: syntax error near unexpected token `fi'
/bin/sh: -c: line 0: `find ./ -not -path "./third_party/*" -name "*.proto" -exec clang-format -i {} \; ; fi'
make: *** [proto-format] Error 2
```

File: Makefile [hunk 1]
```diff
@@ -40,13 +40,15 @@ containerProtoFmt=cosmos-sdk-proto-fmt-$(containerProtoVer)
...
+	@which protoc >/dev/null || (echo "protoc not found. Installing..." && \
+        (uname | grep -q Darwin && brew install protobuf || sudo apt install -y protobuf-compiler))
+	@sh ./scripts/protocgen.sh
 
 proto-format: ## Formats protobuf files
 	@echo "Formatting Protobuf files"
-	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerProtoFmt}$$"; then docker start -a $(containerProtoFmt); else docker run --name $(containerProtoFmt) -v $(CURDIR):/workspace --workdir /workspace tendermintdev/docker-build-proto \
-		find ./ -not -path "./third_party/*" -name "*.proto" -exec clang-format -i {} \; ; fi
+	@which protoc >/dev/null || (echo "protoc not found. Installing..." && \
+        (uname | grep -q Darwin && brew install protobuf || sudo apt install -y protobuf-compiler))
```
Comment (danwt) [inline Makefile:50]: I wonder if the install can be deduped 🤷

**Review by danwt — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (hoangdv2429) [reply Makefile:50]: Yes, we actually can

**Comment by mtsitrin** [comment]
still fails for me
```
make proto-format

Formatting Protobuf files
find: clang-format: No such file or directory
find: clang-format: No such file or directory
```

**Comment by hoangdv2429** [comment]
> still fails for me
> 
> ```
> make proto-format
> 
> Formatting Protobuf files
> find: clang-format: No such file or directory
> find: clang-format: No such file or directory
> ```

do you have clang-format

**Comment by hoangdv2429** [comment]
nvm, I'll make another check for clang-format as well

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**